tests/test_telemetry.py

```
@@ -0,0 +1,77 @@
+        "api-token-secret",
+    }
+
+    @telemetry.track_usage("test.success")
+    def decorated(positional_value: str, **kwargs: str) -> str:
+        assert positional_value in sensitive_values
+        assert kwargs
+        return "ok"
+
+    result = decorated(
+        "positional-secret",
+        password="password-secret",  # pragma: allowlist secret
```

Ruff (0.15.21)

[error] Possible hardcoded password assigned to argument: "password"

(S106)